srcpkgs/librsvg/template

```diff
@@ -14,9 +14,9 @@ maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Projects/LibRsvg"
 # update changelog when release series changes
-changelog="https://gitlab.gnome.org/GNOME/librsvg/-/raw/librsvg-2.57/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/librsvg/-/raw/librsvg-2.58/NEWS"
```

Comment:
could automate this with `${version%.*}`

Reply:
I think the informal policy is to have no variable in `changelog`

Reply:
I'm fine either way but it feels like duplicating the version here is more error-prone

Reply:
@icp1994 please use the variable

Reply:
Are you sure though? There has been strong pushback against using variables in places like homepage, changelog etc by other maintainers in the past.

Reply:
i don't see validity of "strong pushback" claim. The only thing in this regard i can remember was https://github.com/void-linux/void-packages/issues/27735 and yes, it's still unsettled

* `homepage` is usually free of variables for parsing reasons (i think)
* `distfiles` don't make sense without variables (especially `$version`)
* `changelog` is preferred to point to the active branch if there is one. if there is no such thing, the next best thing is to point to the versioned hyperlink.

if not following an idiom (regardless of its justification) leads to less error-prone maintenance overhead, do it.

Reply:
Fine by me then.